test/ml-memleaksfull.graph

```diff
@@ -0,0 +1,17 @@
+perfkeys: Total Allocated Memory:, Total Freed Memory:
+files: ml-memleaksfull.dat, ml-memleaksfull.dat
+ylabel: bytes
+graphname: ml-memusagefull
```

Comment:
I think you copied this from memleaksfull.graph, but `graphname` is deprecated and isn't used/needed.

Reply:
I removed graphname from all memleak graphs